build/templates/_grpc_stub_interpreter.py/default_method.py.mako

```diff
@@ -12,13 +12,18 @@
     if return_statement == 'return':
         return_statement = None
     capture_response = 'response = ' if return_statement else ''
+    included_in_proto = f.get('included_in_proto', True)
```

Comment:
This is a new field generated in the nimi-python functions.py metadata to indicate if the function is accessible through grpc-device / the proto file.

Reply:
This is actually already submitted to ni:master; it's no longer new.